templates/rancher-backup/0/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+backup:
+  environment:
+    CRON_SCHEDULE: ${CRON_SCHEDULE}
+    BACKEND: ${BACKEND}
+    TARGET_PATH: ${TARGET_PATH}
+    BK_FULL_FREQ: ${BK_FULL_FREQ}
+    BK_KEEP_FULL: ${BK_KEEP_FULL}
+    BK_KEEP_FULL_CHAIN: ${BK_KEEP_FULL_CHAIN}
+    VOLUME_SIZE: ${VOLUME_SIZE}
```

Comment:
Autogenerated RANCHER_API_URL, RANCHER_API_KEY and RANCHER_API_SECRET with labels below. Don't need to specify them.